cmd/imgcollector/config/config_test.go

```diff
@@ -0,0 +1,13 @@
+package config
+
+import (
+	"testing"
+	"testing/fstest"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestReadImagePullSecret(t *testing.T) {
+	_, err := ReadImagePullSecret(fstest.MapFS{".dockerconfigjson": {}})
```

Comment:
* Add better input containing some actual values.
* Assert returned value.
* What is the format of dockerconfigjson? Is it base64 encoded string? In such case looks like you are missing base64 decode before unmarshal.

Reply:
Done